protobuf/types/process.proto

```diff
@@ -1,27 +1,43 @@
 syntax = "proto3";
 
+import "gogo/protobuf/gogoproto/gogo.proto";
+
 package types;
 option go_package = "github.com/mesg-foundation/engine/protobuf/types";
 
+
 // A process is a configuration to trigger a specific task when certains conditions of a trigger are valid.
 message Process {
   // Node of the process
   message Node {
     message Result {
       string key = 1;           // Key that identifies the node.
-      bytes instanceHash = 2;  // Hash of the instance that triggers the process.
+
+      // Hash of the instance that triggers the process.
+      bytes instanceHash = 2 [
```

Comment:
missing the `moretags` for all the data. You can find the whole definition here https://github.com/mesg-foundation/engine/blob/2b9afb302adc152576d83a0f8e49811ad1bb5dbb/protobuf/types/process.proto

Reply:
This one will be set (with all validation tags etc) when with PR about removing custom type for event/execution/process.